spec/unit/classes/server_spec.rb

```diff
@@ -157,4 +157,78 @@ class { 'postgresql::globals':
       is_expected.to contain_class('postgresql::repo').with_version('99.5')
     end
   end
+
+  describe 'create config entries' do
+    let(:params) {{ 'config_entries' => {'config_entry' => {'ensure' => 'present', 'name' => 'port_spec', 'value' => '5432'}}}}
+    it 'stops postgresql and changes the port' do
+      is_expected.to contain_file('systemd-override')
```

Comment:
Are these expectations correct?  Shouldn't it check that it's `expected.to contain_postgresql__server__config_entry`?